doc/nimc.rst

```diff
@@ -266,11 +266,11 @@ configuration file should contain something like::
 Cross compilation for Windows
 =============================
 
-To cross compile for Windows from Linux or OSX using the MinGW-w64 toolchain::
+To cross compile for Windows from Linux or macOS using the MinGW-w64 toolchain::
```

Comment:
Mac is a name, should it be capitalized :grey_question: :thinking: `Mac OS`

Reply:
The current official name is "macOS", see https://en.wikipedia.org/wiki/macOS